extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -540,4 +541,8 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		let selectedBook = this.books.find(b => itemPath.toLowerCase().indexOf(b.bookPath.toLowerCase()) > -1);
 		return selectedBook;
 	}
+
+	dispose() {
+		this.dispose();
```

Comment:
Is there a risk of stack overflows here?

Reply:
Actually, why do we need this method here? There's no parent class to call a dispose for, and we aren't doing anything in this method.

Reply:
I was playing around with exposing the extension level objects and added this for my testing, forgot to remove it.